js2-imenu-extras.el

```diff
@@ -35,6 +35,10 @@
 (require 'cl-lib)
 (require 'js2-mode)
 
+(eval-when-compile
+  (when (<= emacs-major-version 26)
+    (require 'subr-x)))
```

Comment:
The version check wasn't really necessary, FWIW.

`when-let` is autoloaded in newer Emacs, but it doesn't hurt to require the package too.